config/samples/core_v1alpha2_manualscalertrait.yaml

```diff
@@ -3,9 +3,9 @@ kind: ManualScalerTrait
 metadata:
   name: manualscalertrait-sample
 spec:
-  replicaCount: 15
+  replicaCount: 3
   workloadRef:
     apiVersion: "core.oam.dev/v1alpha2"
     kind: ""
     name: "example-containerized-workload"
-    uid: "4dd830b0-4adc-4f77-a20f-856e8ce2f847"
\ No newline at end of file
+    uid: "44dcf50f-cbb1-443d-9b8f-7f719cd7a385"
```

Comment:
nits: the convention is add extra blank line for these files

Reply:
Where? before or after?

Reply:
After. The red color error is "no newline at end of file".

![image](https://user-images.githubusercontent.com/1701782/78171011-a54bf380-7408-11ea-8d5d-6162b7e807ac.png)